<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowWidget</class>
 <widget class="QWidget" name="BorrowWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>752</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,11">
     <item>
      <widget class="QGroupBox" name="group_find">
       <property name="title">
        <string>查找</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>编号</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="edit_id">
          <property name="clearButtonEnabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_find">
          <property name="text">
           <string>查找</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="ReaderInfo" name="reader" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="BookWidget" name="book">
      <attribute name="title">
       <string>借书</string>
      </attribute>
     </widget>
     <widget class="BorrowInfoWidget" name="borrow">
      <attribute name="title">
       <string>还书</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ReaderInfo</class>
   <extends>QWidget</extends>
   <header location="global">ReaderInfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BookWidget</class>
   <extends>QWidget</extends>
   <header location="global">BookWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BorrowInfoWidget</class>
   <extends>QWidget</extends>
   <header location="global">BorrowInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
